components/LocaleSwitcher.tsx

```diff
@@ -0,0 +1,38 @@
+import { useRouter } from 'next/router'
+
+import { getFlagLocale } from '../helpers/flags'
+
+interface LocaleSwitcherProps {
+  useFlags?: boolean
+}
+
+function LocaleSwitcher({ useFlags = false }: LocaleSwitcherProps) {
+  const { locales, locale } = useRouter()
+  const router = useRouter()
+  const { pathname, asPath, query } = router
```

Comment:
Creo que esta linea puede ser combinada con la linea 10 ya que ambas deestructuran el mismo objeto.

Reply:
Kevin, you're right! I fixed it on new commits (just merged). Thanks for the comment!